src/components/Tool/FixCommitmentInfo/Contact.tsx

```diff
@@ -357,15 +367,16 @@ const Contact: React.FC<Props> = ({
                                 setFieldValue('status', event.target.value)
                               }
                             >
-                              {Object.values(StatusEnum).map((status) => (
-                                <MenuItem
-                                  value={status}
-                                  key={status}
-                                  data-testid="statusSelectOptions"
-                                >
-                                  {getLocalizedContactStatus(t, status)}
-                                </MenuItem>
-                              ))}
+                              {phases?.map((phase) => [
```

Comment:
It's weird that we're using an array in JSX. Did you mean to use a fragment?

Reply:
I needed that because an empty fragment was causing problems inside the `select`. It was saying `status` might be undefined and in the `select` it wouldn't show the initial status.

Reply:
Does this work? This will skip rendering the fragment if phase is undefined.

```tsx
{phases?.map(
  (phase) =>
    phase && (
      <>
        <ListSubheader key={phase.id}>
          {getLocalizedPhase(t, phase.id)}
        </ListSubheader>
        ,
        {phase.contactStatuses.map((status) => (
          <MenuItem key={status} value={status}>
            {contactStatuses[status]?.translated}
          </MenuItem>
        ))}
      </>
    ),
)}
```

Reply:
No that still doesn't work

Reply:
🤷 OK I guess it's fine as-is then.